src/app/components/pages/PostsIndex.scss

```diff
@@ -212,7 +212,7 @@
     font-family: sans-serif;
   }
   @include MQ(M) {
-    padding: 0.25em 1em;
+    padding: 0em 1em;
```

Comment:
I would prefer not to include style changes in this PR.  Please submit a separate PR with screenshots illustrating the change.